### ISSUE TITLE
training loss curve

### ISSUE THREAD

**Issue by aojunzz** [EleutherAI/math-lm#85]
can you share the log of LLeMA pretraining?

**Comment by zhangir-azerbayev** [comment]
Looking into this, hope to get back to you soon.

**Comment by LegDonn** [comment]
log2 10
